src/app/auth/auth.module.ts

```diff
@@ -0,0 +1,23 @@
+import { NgModule } from '@angular/core';
```

Comment:
did not get the point of auth module and login.module

we need just auth service and login module